### DISCUSSION TITLE
Can we limit how many objects to track in a busy scene?

### DISCUSSION THREAD

**Discussion by saket424** [blakeblackshear/frigate#14970] [Ask A Question]
To limit CPU usage per camera, is it possible to stop tracking new objects in a scene when there are already 5 objects being tracked. New objects don't get tracked till the number of tracked objects drops below 5 as an example

**Comment by NickM-27** [comment]
No, and I'm not sure that would make sense because how can frigate know what object might be the important one to track. 

Your CPU should be capable of handling the load that the system will experience while tracking objects
